node/index.ts

```diff
@@ -91,8 +97,14 @@ const sitemapEntryPipeline = [prepare, sitemapEntry]
 export default new Service<Clients, State, ParamsContext>({
   clients,
   events: {
+    /**
+     * @deprecated This event is being deprecated in favor of the new `/sitemap/apps-routes` endpoint.
+     */
     generateAppsRoutes: [throttle, errors, generationPrepare, generateAppsRoutes],
     generateProductRoutes: [throttle, errors, generationPrepare, tenant, generateProductRoutes, sendNextEvent],
+    /**
+     * @deprecated This event is being deprecated in favor of the new `/sitemap/user-routes` endpoint.
+     */
     generateRewriterRoutes: [throttle, errors, generationPrepare, generateRewriterRoutes, sendNextEvent],
     generateSitemap: [settings, generationPrepare, generateSitemap],
```

Comment:
```suggestion
    generateSitemap: [settings, generationPrepare, generateSitemap],
```

Should we add the same deprecation note to all events? We are not supposed to use them anymore

Reply:
Maybe it makes sense, but it would be interesting to do this at a time when we make a direct get available for these routes as well. I think it can be done when we define how to completely remove the generation.

Reply:
I think that at least for generateSitemap we should remove for this major since this will only be necessary for clients in previous major, allowing it we will give them the possibility to keep calling the generator processes

Reply:
But this generation is still valid for accounts with more than one binding

Reply:
From now on, accounts with more than one binding aren't expected to migrate to this new major. In the future it will, but, this route will not be necessary for them in this case